### PR TITLE
Update package manager to yarn

### PR DESCRIPTION
**Pull request by iuricmp**
This pull request updates the package manager from npm to yarn. This change improves the dependency management and build process of the project.

### CODE REVIEW

File: mobile/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+## Requirements
+
+- Expo Requiments: https://docs.expo.dev/get-started/installation/
+
+## 🚀 How to use
+
+```sh
+# to run on ios:
+yarn ios 
+
```
Comment (jefft0) [inline mobile/README.md:12]: For iOS, is it also necessary to do `pod install` ? If yes, then should put this in the README.

File: mobile/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+## Requirements
+
+- Expo Requiments: https://docs.expo.dev/get-started/installation/
+
+## 🚀 How to use
+
+```sh
+# to run on ios:
+yarn ios
```
Comment (jefft0) [inline mobile/README.md:11]: When I enter `yarn ios`, it prints "expo: command not found". But if I first enter `yarn`, then `yarn ios` build and runs. Should the README way to first enter `yarn`?

File: mobile/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+- Expo Requiments: https://docs.expo.dev/get-started/installation/
+
+## 🚀 How to use
+
+```sh
+# to run on ios:
+yarn ios 
+
+# to run on android:
+yarn android
```
Comment (jefft0) [inline mobile/README.md:14]: Same as for ios above. This prints "expo: command not found". But if I first enter `yarn` then this builds and runs.

File: mobile/README.md (hunk 2 above)
Comment (iuricmp) [reply mobile/README.md]: Hey @jefft0, I hope the latest commit resolves this issue. Thanks.

**Review by jefft0 — APPROVED** [review]
The makefile works to build and run on Android and iOS simulators.

**Pull request merged**